src/lib/y2partitioner/widgets/btrfs_devices.rb

```diff
@@ -147,6 +147,7 @@ def help_for_raid_levels
       #
       # @return [String]
       def help_for_default_raid_level
+        # TRANSLATORS: help text
         _("<p>" \
             "When <b>DEFAULT</b> RAID level is used, Btrfs will select a RAID level depending on " \
```

Comment:
This is not longer DEFAULT in upcase. This would be now more like "When the default RAID level is used,..."